platform/ui/src/contextProviders/ViewportGridProvider.tsx

```diff
@@ -146,6 +146,7 @@ export function ViewportGridProvider({ children, service }) {
           numRows,
           layoutOptions,
           layoutType = 'grid',
+          activeViewportIndex,
```

Comment:
Shouldn't this default to 0?

Reply:
No. Because the previous code was attempting (and I stress attempting) to maintain the previous active viewport for a straight layout change. For example, when in 2x2 and the bottom-right viewport is active. Switching to 3x3 should still maintain that (middle now) viewport active. It wasn't before, it is now. So `undefined`/`null` achieves that. Otherwise you will always typically get the top-left active on a viewport change. In OHIF demo there is strange behaviour whereby the second viewport get activated. I needed to explicitly set the active viewport when toggling off one-up and fixed this behaviour since I was already in the code.